demo-app/src/app/app.module.ts

```diff
@@ -80,15 +80,14 @@ export class AppModule {
 
 		errorMessageService.addErrorMessages({
 			required: "DEMO.FORM_VALIDATION.WITH_NGX_FORM_ERRORS.REQUIRED",
-			// "matchingPasswords.password.required": "DEMO.FORM_VALIDATION.WITH_NGX_FORM_ERRORS.REQUIRED",
+			"matchingPasswords.password.required": "DEMO.FORM_VALIDATION.WITH_NGX_FORM_ERRORS.PASSWORD_REQUIRED",
 			minlength: "DEMO.FORM_VALIDATION.WITH_NGX_FORM_ERRORS.PASSWORD.MIN_LENGTH",
 			maxlength: "DEMO.FORM_VALIDATION.WITH_NGX_FORM_ERRORS.PASSWORD.MAX_LENGTH",
 			pattern: "DEMO.FORM_VALIDATION.WITH_NGX_FORM_ERRORS.PASSWORD.PATTERN"
 		});
 
 		errorMessageService.addFieldNames({
 			username: "DEMO.FIELDS.USER_NAME",
-			"matchingPasswords.password": "DEMO.FIELDS.PASSWORD_ALIAS",
```

Comment:
Why removing this?

Reply:
Ah because it is not used... the alias defined in the template via the `ngxFormErrorsFieldName` attribute takes precedence over this.

I think I'll keep this line but just change the translation to make it clear that is not used at all and why ;)

Reply:
Done... added back this line with a descriptive translation 😉